docs/1 Introduction/3 Quickstart.md

```diff
@@ -0,0 +1,19 @@
+# Quickstart
+
+With `starport` installed on your machine, you can now build your very first blockchain application!
+
+```bash
+starport app github.com/username/myapp
+```
+
+Serve the blockchain application
+
+```bash
+starport serve
+```
+
+Add a new transaction type to your application
```

Comment:
`transaction type`
I wouldn't say it like that, we add a new data structure in the blockchain and transaction handlers to interact with it